Android.mk

```diff
@@ -1,6 +1,16 @@
 LOCAL_PATH:= $(call my-dir)
 
+ifneq (,$(wildcard frameworks/av/media/mediaserver/Android.mk))
```

Comment:
The logic and semantics have changed quite a bit since https://github.com/sailfishos/droidmedia/pull/58/commits/ee18b689aff68bb6b414e3e871e2b9b18adcd790. Is the authorship still Björn's?:)
Asking, because I'll need to port this patch to https://github.com/libhybris/libhybris/pull/467#issuecomment-748519950

Reply:
It seems there was nothing left of the original commit so I changed the author to me.